go.mod

```diff
@@ -217,28 +216,34 @@ replace (
 	github.com/briandowns/spinner => github.com/alonyb/spinner v1.12.7
 	github.com/docker/machine => github.com/machine-drivers/machine v0.7.1-0.20211105063445-78a84df85426
 	github.com/samalba/dockerclient => github.com/sayboras/dockerclient v1.0.0
-	k8s.io/api => k8s.io/api v0.21.2
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.2
-	k8s.io/apimachinery => k8s.io/apimachinery v0.21.2
-	k8s.io/apiserver => k8s.io/apiserver v0.21.2
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.21.2
-	k8s.io/client-go => k8s.io/client-go v0.21.2
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.21.2
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.21.2
-	k8s.io/code-generator => k8s.io/code-generator v0.21.2
-	k8s.io/component-base => k8s.io/component-base v0.21.2
-	k8s.io/component-helpers => k8s.io/component-helpers v0.21.2
-	k8s.io/controller-manager => k8s.io/controller-manager v0.21.2
-	k8s.io/cri-api => k8s.io/cri-api v0.21.2
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.21.2
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.21.2
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.21.2
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.21.2
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.21.2
-	k8s.io/kubectl => k8s.io/kubectl v0.21.2
-	k8s.io/kubelet => k8s.io/kubelet v0.21.2
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.21.2
-	k8s.io/metrics => k8s.io/metrics v0.21.2
-	k8s.io/mount-utils => k8s.io/mount-utils v0.21.2
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.2
+	k8s.io/api => k8s.io/api v0.22.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.22.5-rc.0
+	k8s.io/apiserver => k8s.io/apiserver v0.22.4
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.22.4
+	k8s.io/client-go => k8s.io/client-go v0.22.4
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.22.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.22.4
+	k8s.io/code-generator => k8s.io/code-generator v0.22.5-rc.0
+	k8s.io/component-base => k8s.io/component-base v0.22.4
+	k8s.io/component-helpers => k8s.io/component-helpers v0.22.4
+	k8s.io/controller-manager => k8s.io/controller-manager v0.22.4
+	k8s.io/cri-api => k8s.io/cri-api v0.23.0-alpha.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.22.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.22.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.22.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.22.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.22.4
+	k8s.io/kubectl => k8s.io/kubectl v0.22.4
+	k8s.io/kubelet => k8s.io/kubelet v0.22.4
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.22.4
+	k8s.io/metrics => k8s.io/metrics v0.22.4
+	k8s.io/mount-utils => k8s.io/mount-utils v0.22.5-rc.0
```

Comment:
And do we want to be updating to RC versions?

Reply:
hm.... I think you are right, for simplicity we could just bump everything to 22.4

Reply:
done